api/ver2/utils/validators.py

```diff
@@ -0,0 +1,48 @@
+from flask_jwt_extended import (get_jwt_identity)
+from api.ver1.utils import error
+from api.ver2.utils.strings import status_401, admin_key
+from api.ver2.models.users import Users
+from api.strings import id_key
+import re
+
+
+def is_bool(*args):
+    for arg in args:
+        if not isinstance(arg, bool):
+            return False
+    return True
+
+
+def is_string(*args):
+    for arg in args:
+        if not isinstance(arg, str):
+            return False
+    return True
+
+
+def is_number(*args):
+    for arg in args:
+        if not isinstance(arg, int) or isinstance(arg, float):
+            return False
+    return True
+
+
+def is_not_admin():
+    user = Users().get_by(id_key, get_jwt_identity())
+
+    if not user[admin_key]:
+        return error(
+            "Forbidden: This action is reserved to Admins only", status_401)
```

Comment:
Consider revising this to `Unauthorized` in place of `Forbidden`. Unless you feel that that is the best way to put it.

Reply:
OK...Will check on that